tests/utils.go

```diff
@@ -1243,39 +1236,6 @@ func deployOrWipeTestingInfrastrucure(actionOnObject func(unstructured.Unstructu
 			PanicOnError(err)
 		}
 	}
-	// Scale KubeVirt back
-	err, _ = DoScaleDeployment(flags.KubeVirtInstallNamespace, "virt-api", replicasApi)
-	PanicOnError(err)
-	err, _ = DoScaleDeployment(flags.KubeVirtInstallNamespace, "virt-controller", replicasController)
-	PanicOnError(err)
-	_, _, newGeneration, err := DoScaleVirtHandler(flags.KubeVirtInstallNamespace, "virt-handler", selector)
```

Comment:
so, this function is
- scaling control plane down
- installing some dependencies
- then starting up the control plane again

That's being done to ensure the control plane components pick up on the dependencies. Are we certain this isn't needed anymore?

Reply:
@davidvossel Hi. From what I've seen in the original [PR](https://github.com/kubevirt/kubevirt/pull/931) that added this scale logic, the intention was so that the virt components will pick the updated Kubevirt configmap after the installation of the dependencies. I could not spot any testing infra resource that could touch the Kubevirt configmap. I can leave this logic as is and just drop the scale back step (which is the problematic one) since now it will be down by the virt-operator reconciler

Reply:
i think this change should be okay. The kubevirt CR now supports dynamic updates and dependencies like CDI should be dynamic as well